composer.json

```diff
@@ -11,11 +11,11 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "dragonbe/vies": "^2.1",
-        "symfony/http-kernel": "^3.0||^4.0",
-        "symfony/dependency-injection": "^3.0||^4.0",
-        "symfony/validator": "^3.0||^4.0"
+      "php": ">=7.2.5",
+      "dragonbe/vies": "^2.1",
+      "symfony/http-kernel": "^5.0",
```

Comment:
Do we have to drop SF 3/4 support, or can we still support all of them?

Reply:
we could support 3.4 and 4.4 since they are still maintained by Symfony. We could also backports changes to the 1.x branch if we need to and just focus on 5 for this release.

Reply:
I would suggest to at least keep SF4 support. Upgrading the monolith to SF5 will probably take a while, and keeping SF4 will prevent us from doing backports. Dropping SF3 is fine with me then.